src/DevChatter.Bot.Core/Games/Hangman/HangmanGame.cs

```diff
@@ -130,12 +130,16 @@ public void AskAboutLetter(IChatClient chatClient, string letterToAsk, ChatUser
 
                 chatClient.SendMessage($"Yep, {letterToAsk} is in here. {MaskedPassword}");
             }
-            else
+            else if (ALL_LETTERS.Contains(letterToAsk))
             {
                 chatClient.SendMessage($"No, {letterToAsk} is not in the word.");
                 _hangmanDisplayNotification.HangmanWrongAnswer();
                 CheckForGameLost(chatClient);
             }
+            else
```

Comment:
I like it!

Perhaps move this before the `_guessedLetters.Add(new HangmanGuess(letterToAsk, chatUser));` line and return, so we don't add this invalid choice to the collection. That collection is how we're determining a lost game.

Reply:
Good catch, I will fix this later today.